### DISCUSSION TITLE
Is there a user manual somewhere?

### DISCUSSION THREAD

**Discussion by marktilles** [Jyers/Marlin#465] [Q&A] [answered]
Hi, I'm new to GitHub and have successfully loaded the latest version 1.3.0 on my Ender 3 V2.   I can't locate ay sort of user manual anywhere, does such a manual exist?
Thanks in advance, 
Mark

**Comment by t3chguy** [comment, marked as answer]
https://github.com/Jyers/Marlin/wiki/Primary-Features and other pages in the Wiki

**Reply by marktilles** [reply]
Thank you! Where might I look for descriptions of the many other options and settings for newbies to the Ender 3d printer world?
